<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2018-10-13T15:00:28Z" name="JSErrorTestSuite" finished-at="2018-10-13T15:00:38Z" duration-ms="9615">
    <groups>
    </groups>
    <test started-at="2018-10-13T15:00:28Z" name="JSErrorTests" finished-at="2018-10-13T15:00:38Z" duration-ms="9615">
      <class name="com.herokuapp.theinternet.jserrortests.JSErrorTests">
        <test-method is-config="true" signature="setUp(java.lang.reflect.Method, java.lang.String, org.testng.ITestContext)[pri:0, instance:com.herokuapp.theinternet.jserrortests.JSErrorTests@6e509ffa]" started-at="2018-10-13T15:00:28Z" name="setUp" finished-at="2018-10-13T15:00:37Z" duration-ms="8348" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.herokuapp.theinternet.jserrortests.JSErrorTests.jsErrorTest()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@34dc85a]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="jsErrorTest()[pri:0, instance:com.herokuapp.theinternet.jserrortests.JSErrorTests@6e509ffa]" started-at="2018-10-13T15:00:37Z" name="jsErrorTest" finished-at="2018-10-13T15:00:37Z" duration-ms="523" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Severe error: http://the-internet.herokuapp.com/javascript_error 6:51 Uncaught TypeError: Cannot read property 'xyz' of undefined]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Severe error: http://the-internet.herokuapp.com/javascript_error 6:51 Uncaught TypeError: Cannot read property 'xyz' of undefined
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.herokuapp.theinternet.jserrortests.JSErrorTests.jsErrorTest(JSErrorTests.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jsErrorTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.herokuapp.theinternet.jserrortests.JSErrorTests@6e509ffa]" started-at="2018-10-13T15:00:37Z" name="tearDown" finished-at="2018-10-13T15:00:38Z" duration-ms="713" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.herokuapp.theinternet.jserrortests.JSErrorTests -->
    </test> <!-- JSErrorTests -->
  </suite> <!-- JSErrorTestSuite -->
</testng-results>
